### ISSUE TITLE
Give command borks out with Error if player is not online

### ISSUE THREAD

**Issue by MODNMETL** [EssentialsX/Essentials#5829]
### Type of bug

Error in console

### `/ess dump all` output

https://essentialsx.net/dump.html?id=96e21f7d189e4c3480d6777965f57a67

### Error log (if applicable)

https://paste.md-5.net/velapawaru.md

### Bug description

When using the essentials give command from the console if the player is online we get a nicely formatted message output confirming what's just been given to whom and how much. But if we issue the same command when the player is offline the command throws a ton of errors (when ess debug is enabled) and the console message is just "Error: Player not found." 

### Steps to reproduce

Use the give command from the console to give a player an item who is not online while running 2.21.0-dev+93-3a6fdd9 (much older snapshot versions didn't have this problem btw)

### Expected behaviour

For the console message to report back that the player is offline as I'm sure it used to (or if it was still 'not found' that it be a properly handled info message instead of it throwing all those debug warnings)

### Actual behaviour

Falls out of the command being called due to being unable to complete in a controlled manner

**Comment by MODNMETL** [comment]
Actually I've just checked and the last snapshot I'm aware of where this wasn't an issue was EssentialsX-2.21.0-dev+24-0af4436

**Comment by Evidentsinger14** [comment]
You’re saying you get stacktraces when debug is enabled? If so, that’s intended. That’s why it’s in debug.
You should only get the “Error: Player not found” message when debug is disabled.

**Comment by MODNMETL** [comment]
> You’re saying you get stacktraces when debug is enabled? If so, that’s intended. That’s why it’s in debug. You should only get the “Error: Player not found” message when debug is disabled.

Yeah I only included the debug info because I didn't think it threw the same on dev24 as a plugin we use which watches for these give events worked then and not now. But I've just rolled back to Essentials dev24 and the debug is the same except it doesn't have the new minimessage colour coding for the Error: Player not found. so clearly we need to fix the plugin which watches for it so that it handles the new minimessage format correctly.

I'll close this issue
